server/tests/employer.site.e2e.test.js

```diff
@@ -29,19 +29,54 @@ const siteObject = ({ id, name }) => ({
   siteContactEmail: 'test.site@hcpa.fresh',
 });
 
+/**
+ * This method is required because different endpoints for creating/editing
+ * sites have different names for similar fields (ie: siteContactPhone vs siteContactPhoneNumber)
+ *
+ * This method wraps siteObject() and renames those fields to pass validation
+ *
+ * @param {*} object containing an id and name used to create a test site
+ * @returns an object containing the given id and name and the rest of the required fields for batch saving
+ */
+const batchSiteObject = ({ id, name }) => {
```

Comment:
Do you think it would be possible to rename the attributes in the different places so that they match? Or are there constraints (other than time, resources) preventing us from doing so?

If it's possible, I'm in favour of putting a ticket in the backlog to clean that up.

Reply:
It looks like a decent sized chunk of understanding/work to do, I agree it would be good to add something to the backlog

- siteContactPhone: 16 results
- siteContactPhoneNumber: 10 results
- operatorPhone: 34 results